Sources/DateExtensions.swift

```diff
@@ -8,27 +8,38 @@
 
 import Foundation
 
+/// EZSE: This Dict help to cache already created formatter to use them in future, helps in performace improvement.
+fileprivate var dateFormatterDict: [String:DateFormatter] = [:]
```

Comment:
This entity needs synchronization.

Reply:
I have updated the pull request kindly review @Khalian

Reply:
I think this is now synchronized, and is ready to be merged.